test/unit_test/common/test_token_utils_lazy_encoder.py

```
@@ -0,0 +1,99 @@
+
+        module = _reimport_token_utils(monkeypatch, unavailable)
+
+        with pytest.raises(RuntimeError, match="BPE table unavailable"):
+            module.num_tokens_from_string("hello")
+
+    def test_unknown_attribute_still_raises(self, monkeypatch):
+        """The module __getattr__ must not mask genuine typos."""
+        module = _reimport_token_utils(monkeypatch, lambda name: None)
+
+        with pytest.raises(AttributeError):
+            module.no_such_name
```

Ruff (0.16.2)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)